60-steam-input.rules

```diff
@@ -60,9 +60,15 @@ KERNEL=="hidraw*", ATTRS{idVendor}=="0e6f", ATTRS{idProduct}=="0188", MODE="0660
 # Nacon PS4 Revolution Pro Controller
 KERNEL=="hidraw*", ATTRS{idVendor}=="146b", ATTRS{idProduct}=="0d01", MODE="0660", TAG+="uaccess"
 
+# Nacon Daija PS4 Arcade Stick
+KERNEL=="hidraw*", ATTRS{idVendor}=="146b", ATTRS{idProduct}=="0d09", MODE="0660", TAG+="uaccess"
```

Comment:
SDL already knows about this as a PS4 controller clone in `src/joystick/controller_type.c`, but not in `src/joystick/SDL_gamepad_db.h`.

Reply:
On PS4 mode steam detects the controller, but the buttons are not mapped correctly. It thinks:

❌ -> ⭕
🔳  -> ❌
🔺 -> 🔳
⭕ -> nothing

With this change, the buttons on PS4 work correctly.

Reply:
I'm surprised this change makes any difference to you, because https://github.com/ValveSoftware/steam-devices/pull/34#issuecomment-1579709226 doesn't include a `raw-hid` device for the Daija, which would suggest that there's no `hidraw` device node for this rule to affect.

But maybe that was a mistake in the info provided in that comment?